### ISSUE TITLE
Reverse text broken

### ISSUE THREAD

**Issue by ranebrown** [so-fancy/diff-so-fancy#410]
Reverse text does not seem to be working correctly on the latest version (see below settings). The text shows as a solid green block. Works as expected on 1.3.0.

```config
[color "diff-highlight"]
	oldNormal = red bold
	oldHighlight = red bold reverse
	newNormal = green bold
	newHighlight = green bold reverse
```

**Comment by scottchiefbaker** [comment]
I just landed a fix for that in 27ef4604e1046d72f12edf34d4266da549e55a8e. Can you text the latest `next` branch and see if that fixes it?

**Comment by ranebrown** [comment]
That seems to work again. Thanks!
